.github/workflows/codeql-analysis.yml

```diff
@@ -37,12 +37,6 @@ jobs:
         # Learn more about CodeQL language support at https://git.io/codeql-language-support
 
     steps:
-    - name: Set up Go 1.x
-      uses: actions/setup-go@v5
-      with:
-        go-version: ^1.18
```

Comment:
`go-version: 1.23.1` may work?

Reply:
no. codeql is built with 1.22. It doesn't support go 1.23.

Reply:
but codeql is still failing: https://github.com/kubernetes-sigs/blob-csi-driver/actions/runs/10797718254/job/29949578807?pr=1588